requirements.tests.txt

```diff
@@ -3,6 +3,7 @@ pytest==4.4.1
 freezegun==0.3.11
 pytest-cov==2.6.1
 pytest-postgresql==1.4.0
+mirakuru==1.1
```

Comment:
Mirakuru was an unpinned dependency of pytest-postgresql, and version 2 drops py2 support, so pin it